docs/upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-consistent-topology.rst

```diff
@@ -3,4 +3,102 @@ Enable Consistent Topology Updates
 =====================================
 
 This article explains how to enable consistent topology changes
-when you upgrade from version 5.4 to 6.0.
\ No newline at end of file
+when you upgrade from version 5.4 to 6.0.
+
+Introduction
+============
+
+ScyllaDB Open Source 6.0 introduces :ref:`consistent topology changes based on Raft <raft-topology-changes>`.
+It is enabled automatically for newly created clusters, but - unlike in the case of schema managed on Raft -
+consistent topology changes are *not* automatically enabled for clusters that were upgraded from an older version.
+In those clusters, consistent topology changes need to be enabled with a dedicated upgrade procedure.
+
+Before running the procedure, you **must** check that the cluster meets some prerequisites
+and you **must** ensure that some administrative procedures will not be run
+while the upgrade procedure is in progress.
+
+.. _enable-raft-topology-6.0-prerequisites:
+
+Prerequisites
+=============
+
+* Make sure that all nodes in the cluster are upgraded to ScyllaDB Open Source 6.0.
+* Verify that :ref:`schema on raft is enabled <schema-on-raft-enabled>`.
+* Make sure that all nodes enabled ``SUPPORTS_CONSISTENT_TOPOLOGY_CHANGES`` cluster feature.
+  One way to verify this is to look for the following message in the log:
+
+  .. code-block:: none
+
+    features - Feature SUPPORTS_CONSISTENT_TOPOLOGY_CHANGES is enabled
+
+  Alternatively, this can be checked programmatically by checking whether ``value`` column under the key ``enabled_features`` contains the name of the feature in the ``system.scylla_local`` table. This can be done with the following query:
+
+  .. code-block:: none
```

Comment:
perhaps additionally we could add full bash example? It'd definitely save admins some effort

Reply:
Hmm, perhaps I can include a snippet like `cqlsh -e "...query string..."`, like we do with `curl` for the HTTP endpoint.

Reply:
Ah, right, this is not only about querying but also checking whether the feature name is there. It makes even more sense to provide a bash command

Reply:
I guess the first user of this will be cloud which'd copy it into siren codebase